ProjectVoyage/View Controllers/Chat list/ChatListVC.m

```diff
@@ -0,0 +1,93 @@
+//
+//  ChatListVC.m
+//  ProjectVoyage
+//
+//  Created by Gui David on 7/6/22.
+//
+
+#import "ChatListVC.h"
+#import "Chat.h"
+#import "ChatCell.h"
+#import "MessagesVC.h"
+#import "LoginVC.h"
+
+@interface ChatListVC () <UITableViewDataSource, UITableViewDelegate>
+
+@property (nonatomic, strong) IBOutlet UIBarButtonItem *createChatBtn;
+@property (nonatomic, strong) IBOutlet UITableView *tableView;
+
+@property (nonatomic, strong) UIRefreshControl *refreshControl;
+@property (nonatomic, strong) NSArray *chats;
+
+@end
+
+@implementation ChatListVC
+
+#pragma mark - Initialization
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    self.tableView.delegate = self;
+    self.tableView.dataSource = self;
+    
+    [self initRefreshControl];
+}
+
+- (void) viewWillAppear:(BOOL)animated {
+    [self refreshHomeFeed:self.refreshControl];
```

Comment:
when overriding these methods, be sure to call implementations of super class

Reply:
Could you provide more information? I'm curious to internalize the learning here